misc/userscripts/readability

```diff
@@ -21,20 +21,35 @@ tmpfile = os.path.join(
 if not os.path.exists(os.path.dirname(tmpfile)):
     os.makedirs(os.path.dirname(tmpfile))
 
+# styling for window size depended margin scaling and line height
+header = """<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>%s</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <style type="text/css">
+       body{margin:40px auto;max-width:650px;line-height:1.4;padding:0 10px}
+       h1,h2,h3{line-height:1.2}
```

Comment:
Please add some whitespace in the CSS - lines are free and it's make it much more readable :wink:

Something like:

```css
body {
    margin: 40px auto;
    max-width: 650px;
    ...
}
```

(also, the style is indented by three spaces while you use four for the HTML - I guess that's unintended?)